isotpsniffer.c

```diff
@@ -79,6 +79,7 @@ void print_usage(char *prg)
 	fprintf(stderr, "         -f <format>  (1 = HEX, 2 = ASCII, 3 = HEX & ASCII - default: %d)\n", FORMAT_DEFAULT);
 	fprintf(stderr, "         -L           (set link layer options for CAN FD)\n");
 	fprintf(stderr, "         -h <len>     (head: print only first <len> bytes)\n");
+	fprintf(stderr, "         -q           (don't quit on read error, allows to receive malformed frames)\n");
```

Comment:
I would suggest to use '-i' analog to cangen:

-i            (ignore syscall errors to receive malformed PDUs)